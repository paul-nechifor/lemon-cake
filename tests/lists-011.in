(list (list 1 'df' (list 'asf') () (list 'bb' ) ) 123)
