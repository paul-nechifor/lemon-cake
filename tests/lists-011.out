((1 'df' ('asf') () ('bb')) 123)
